### PR TITLE
Update government shutdown faq.mdx

### PR DESCRIPTION
**Pull request by A-L33**
Updating the answer to "Do I still have to pay my individually billed travel card bill?" for the government shutdown FAQ.

### CODE REVIEW

**Comment by A-L33** [comment]
@rebekahperillo I drafted this PR to reflect the changes to the bill pay question.  Please review and if you are ok with it, we'll send it to Ryan for review as I had to make some format decisions that I wanted to run by him.

File: src/content/faq/faq.mdx [hunk 1]
```diff
@@ -443,22 +443,9 @@ No, it is never appropriate to use your GSA SmartPay account for expenses not re
...
-- Take appropriate action to prevent accounts from becoming delinquent, suspended or canceled.
-- Ensure finance charges are not assessed.
-- Waive all applicable late payment fees.
-- Continue to generate account statements.
-- Work with agencies to maintain risk mitigation measures.
-- Coordinate with agencies and account holders to bring accounts up to date once funding becomes available.
-
-The banks use highly automated account management processes that may require manual intervention by bank staff during a shutdown. As a result, isolated instances of errant activities such as the issuance of late payment notices on an account affected by the shutdown can occur. Should this happen, please follow your agency procedures or contact your servicing contractor bank as appropriate.
+Citibank: For agencies, organizations, and card/accountholders unable to make payments in the event of a government-wide shut down, Citibank will ensure that during such periods accounts will not age delinquent or be suspended or cancelled and finance charges will not be assessed. The bank will continue to generage Statements of Account.
```
Comment (brentryanjohnson) [inline src/content/faq/faq.mdx:446]: Consider splitting `card/accountholders` to `card/account holders` (add space).

See [style guide for hyphen use](https://github.com/GSA/smartpay-website/wiki/GSA-SmartPay%C2%AE-%7C-Style-Guide-for-the-Website-and-Training-Platform#hyphens) (should be `governmentwide`).

Consider rewording `age delinquent` to a more plain language version.

File: src/content/faq/faq.mdx [hunk 2]
```diff
@@ -443,22 +443,9 @@ No, it is never appropriate to use your GSA SmartPay account for expenses not re
...
 
-For accounts that were past due prior to the start of the shutdown, routine account management and payment rules apply.
-
-Agencies may issue supplemental account holder guidance regarding individually billed account management and payments during a shutdown, consistent with applicable GSA SmartPay contract and task order terms.
+U.S. Bank: Accounts could become past due until budgetary issues are resolved, and as such, no accounts will be deactivate without authorization from the GSA contracting officer. U.S. Bank will work with the government to implement risk mitigation measures to monitor fraud and unusual delinquencies, as well as to bring past due accounts up to date once the budget is approved.
```
[the remark this answers is not in the document]
Comment (rebekahperillo) [reply src/content/faq/faq.mdx:448]: Also, looks like it should be "no accounts will be deactivated" (missing the -d at the end).

**Comment by A-L33** [comment]
@brentryanjohnson Ready for review!

**Review by brentryanjohnson — APPROVED** [review]

**Pull request merged**